lib/galaxy/webapps/tool_shed/api/repositories.py

```diff
@@ -86,8 +86,36 @@ def add_repository_registry_entry( self, trans, payload, **kwd ):
             % ( name, owner )
         return response_dict
 
+    @expose_api_anonymous_and_sessionless
+    def get_latest_metadata( self, trans, id, **kwd ):
```

Comment:
Reposting explanation I provided offline:

This endpoint is used to populate the "tools or package verified" column in the browse category page.